test_hector.sh

```diff
@@ -31,15 +31,21 @@ exit_loudly () {
    exit 1
 }
 
-
 # Run all INI files (the basic SSPs)
 echo "---------- Running: basic SSPs ----------"
 find ./inst/input -name "*.ini" -print0 | xargs -0 -n1 $HECTOR
 
+# Permafrost (off by default in v3) runs
+echo "---------- Running: permafrost ----------"
```

Comment:
So this test checks to make sure can run without error?

Reply:
Right.